openbb_platform/providers/yfinance/openbb_yfinance/models/currency_historical.py

```diff
@@ -48,12 +48,22 @@ def transform_query(
         params: Dict[str, Any]
     ) -> YFinanceCurrencyHistoricalQueryParams:
         """Transform the query."""
+
         transformed_params = params
-        transformed_params["symbol"] = (
-            f"{transformed_params['symbol'].upper()}=X"
-            if "=X" not in transformed_params["symbol"].upper()
-            else transformed_params["symbol"].upper()
+
+        new_symbols = []
+        symbols = (
+            params["symbol"].split(",")
+            if "," in params["symbol"]
+            else [params["symbol"]]
         )
+        for symbol in symbols:
+            new_symbol = (
+                f"{symbol.upper()}=X" if "=X" not in symbol.upper() else symbol.upper()
+            )
+            new_symbols.append(new_symbol)
```

Comment:
you could replace this whole loop with a single list comprehension:

new_symbols = [f"{s.upper()}=X" if "=X" not in s.upper() else s.upper() for s in symbols]

Reply:
replaced